### PR TITLE
remove mustBeOnlyMailbox as mandatory attribute

### PR DESCRIPTION
**Pull request by dbenchi**

### CODE REVIEW

File: lib/models/CreateMailboxAck.js [hunk 1]
```diff
@@ -21,7 +21,6 @@ export default class CreateMailboxAck extends Model {
     super(jmap);
     Utils.assertRequiredParameterIsObject(response, 'response');
     Utils.assertRequiredParameterIsPresent(response.id, 'response.id');
-    Utils.assertRequiredParameterHasType(response.mustBeOnlyMailbox, 'response.mustBeOnlyMailbox', 'boolean');
 
     this.id = response.id;
     this.mustBeOnlyMailbox = response.mustBeOnlyMailbox;
```
Comment (ddolcimascolo) [inline lib/models/CreateMailboxAck.js]: `|| false` and add a test to verify it.

Comment (dbenchi) [inline lib/models/CreateMailboxAck.js]: Done

**Comment by ddolcimascolo** [comment]
:+1:

**Comment by thomas-hilaire** [comment]
:+1:

**Pull request merged**